### PR TITLE
checkbox alert box, checkbox remembers from parse

### PR DESCRIPTION
**Pull request by pooja335**
positive reinforcement!

### CODE REVIEW

**Comment by kapil1garg** [comment]
LGTM

File: app/xlist/scripts/HomeController.js [hunk 1]
```diff
@@ -125,5 +125,25 @@ angular
       });
     };
 
+    $scope.congratsAlert = function(task) {
+      task.save({
+        done: !task.attributes.done
```
Comment (katsuya94) [inline app/xlist/scripts/HomeController.js]: Can we stick with using `task.get{'done')`, so we don't encounter any more weird problems accessing `attributes` directly.

File: app/xlist/scripts/HomeController.js [hunk 2]
```diff
@@ -125,5 +125,25 @@ angular
...
+      }, {
+        success: function(results) {
+          if (task.attributes.done) {
```
Comment (katsuya94) [inline app/xlist/scripts/HomeController.js]: ditto

**Comment by katsuya94** [comment]
LGTM

**Pull request merged**